main.nf

```diff
@@ -259,6 +260,31 @@ if (params.pseudo_aligner == 'salmon') {
     }
 }
 
+skip_rsem = params.skip_rsem
+if (!params.skipAlignment && !params.skip_rsem && params.aligner != "star") {
+    skip_rsem = true
+    println "RSEM only works with STAR. Disabling RSEM."
```

Comment:
According to the release notes for RSEM  1.3.3 (https://deweylab.github.io/RSEM/updates.html)  there is now a hisat option available.

Reply:
I wasn't aware of that. Would indeed be a nice addition. 
However I won't have time to look into this any time soon. 

Therefore, I opt for merging it now, and adding RSEM+Hisat support in a second PR.

Reply:
Noted - merging this PR now.